public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -542,7 +542,7 @@ table
 // #docregion property-binding-7
 :marked
   ### Binding target
-  a id between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
+  An element property between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
```

Comment:
I thought to add the **attribute** word but then it came to my mind that Angular sets DOM properties and not attributes, thus it will be more clear to the user reading this.